### PR TITLE
Fix notification audio

### PR DESCRIPTION
**Pull request by bronzdoc**
Added postinstall script to replace the wrong version of libffmpegsumo.so in "nw" directory.
https://github.com/slack-for-linux/slack-for-linux/issues/22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
   ],
   "scripts": {
     "start": "node ./run.js",
-    "test": "grunt lint"
+    "test": "grunt lint",
+    "postinstall": "cp libraries/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so"
```
Comment (twolfson) [inline package.json]: I imagine we could get more complex with `postinstall` over time. Can we relocate it to a `postinstall.sh` with the following template:

```bash
#!/usr/bin/env bash
# Exit on first error and output comments
set -e
set -x

# Copy over libffmpegsumo.so
cp libraries/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
```

**Comment by twolfson** [comment]
While it's an interesting proposition to store the compiled binary in our repo, I would prefer not to as it could vary from computer to computer. Can we perform the same `cp` as listed in `nw.js'` documentation (i.e. from `/opt/google/chrome`)?

https://github.com/nwjs/nw.js/wiki/Using-MP3-&-MP4-%28H.264%29-using-the--video--&--audio--tags.#linux

If we are concerned about people not having Google Chrome installed, then maybe we do a `test -f` for the file and if it isn't found, then we tell them to reinstall once `google-chrome` is installed?

```bash
if test -f /opt/google/chrome/libffmpegsumo.so; then
  cp ...
else 
  echo "Couldn't locate libffmpegsumo.so during installation. Audio notifications will be disabled" 1>&2
  echo "To repair this, please install Google Chrome and reinstall \`slack-for-linux\`" 1>&2
fi
```

**Comment by twolfson** [comment]
Also, by adding in `libffmpegsumo.so`, we need to add a note about how we are using GPL code but one of us will take care of that upon landing.

**Comment by twolfson** [comment]
Thanks for taking the time to take care of this issue by the way, we really appreciate it =)

**Comment by bronzdoc** [comment]
Anytime =) 
I made the changes you suggested hope it helps, let me know if there's are any more changes you want me to do.

**Comment by twolfson** [comment]
Looks good. Can you squash the commits into 1 commit for landing?

I think I will add a note to `COLLABORATING.md` about having that as part of PRs. Sorry it wasn't written down anywhere =(

**Comment by bronzdoc** [comment]
All is squashed into one commit now.

**Comment by twolfson** [comment]
Sweet, thanks =)

**Comment by twolfson** [comment]
This has been merged/released in `1.6.0`. Thanks again for this PR =D

**Comment by twolfson** [comment]
Ugh, I typically use CLI based merged for GitHub PRs and as a result this commit body isn't in the CHANGELOG. /cc @wlaurance

**Comment by twolfson** [comment]
Alright, that was a mess to clean up. I had to move back the commit and re-land it via a merge commit (`--no-ff`) with the same markup that GitHub uses. However, it's patched and working now.

https://github.com/lalitkapoor/github-changes/blob/15dd41d8d7ca9e3a9ad0786fc611cef5e7b3760f/bin/index.js#L383-L388

Attribution for bronzdoc is visible under `1.6.1` :tada:

**Pull request merged**